src/commonMain/kotlin/fr/acinq/lightning/crypto/KeyManager.kt

```diff
@@ -47,14 +54,16 @@ interface KeyManager {
      */
     fun newFundingKeyPath(isFunder: Boolean): KeyPath
 
+    fun channelKeys(fundingKeyPath: KeyPath) : ChannelKeys
+
     /**
      *
      * @param tx        input transaction
      * @param publicKey extended public key
```

Comment:
The documentation for all these functions should be updated.

Reply:
done in https://github.com/ACINQ/lightning-kmp/pull/255/commits/3b01af57bbb8bc95af625ce1d11ae54e433214de